src/main/java/model/Order.java

```diff
@@ -74,21 +83,96 @@ public double getTotalPrice() {
                 this.orderItemList.stream().mapToDouble(Item::getPrice).sum() * (1 + SERVICE_CHARGE) * (1 + GST)));
     }
 
-    public String getReceipt(double discount)  {
-        StringBuilder receiptBuilder = new StringBuilder();
-        Set<String> processedItems = new HashSet<>();
+    /**
+     * Returns a formatted receipt header (top part of the receipt)
+     *
+     * @param headerBuilder the StringBuilder to append the header to
+     * @return the formatted receipt header
+     */
+    private StringBuilder getReceiptHeader(StringBuilder headerBuilder) {
+        return headerBuilder.append(ROW_DELIMITER)
+                .append(TITLE)
+                .append(ROW_DELIMITER)
 
-        receiptBuilder.append(ROW_DELIMITER)
+                .append("| ")
                 .append(restaurantName)
-                .append("\n")
+                .append(" ".repeat(ROW_DELIMITER.length() - restaurantName.length() - SUBTITLE_OFFSET))
+                .append("|\n")
+
+                .append("| ")
                 .append(restaurantAddress)
-                .append("\n")
-                .append(ROW_DELIMITER)
-                .append(TITLE)
-                .append(ROW_DELIMITER)
+                .append(" ".repeat(ROW_DELIMITER.length() - restaurantAddress.length() - SUBTITLE_OFFSET))
+                .append("|\n")
+
+                .append("| Order Type: ")
```

Comment:
Would it be better if the restaurant name + restaurant address is formatted in the centre at the top of the receipt? (Because that's how most receipts are formatted now) Just a suggestion, because I think the current format looks great too!